src/skins/constants.tsx

```diff
@@ -1,5 +1,6 @@
 export const MOVISTAR_SKIN = 'Movistar';
 export const O2_SKIN = 'O2';
+export const O2_NEW_SKIN = 'O2';
```

Comment:
I'd call it `'O2-new'`, like we did with `'Vivo-new'`

Reply:
oh, sure. that was a mistake